modules/web-console/web-console-common/src/main/java/org/apache/ignite/console/websocket/WebSocketEvent.java

```diff
@@ -105,14 +104,33 @@ public String getPayload() {
 
     /**
      * @param payload New payload.
-     * @return {@code this} for chaining.
      */
-    public WebSocketEvent setPayload(String payload) {
+    public void setPayload(String payload) {
         this.payload = payload;
+    }
 
-        return this;
+    /**
+     * Constructor with auto generated ID.
+     *
+     * @param payload Payload.
+     */
+    public WebSocketEvent withPayload(Object payload) {
+        return new WebSocketEvent(this.reqId, evtType, toJson(payload));
     }
 
+    /**
+     * Constructor with auto generated ID.
```

Comment:
Wrong javadoc. It is better to write: "Create event with error for response with same ID".